pkg/operator/staticpod/controller/controller.go

```diff
@@ -0,0 +1,61 @@
+package controller
+
+import (
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openshift/library-go/pkg/operator/staticpod/controller/common"
+	"github.com/openshift/library-go/pkg/operator/staticpod/controller/deployment"
+	"github.com/openshift/library-go/pkg/operator/staticpod/controller/installer"
+	"github.com/openshift/library-go/pkg/operator/staticpod/controller/node"
+)
+
+type StaticPodOperator struct {
```

Comment:
controller or operator? package is controller.

Reply:
isn't operator just a lightweight controller? ;-)

Reply:
or is an operator a set of controllers?

Reply:
Should this be one package level up?

Reply:
per slack: move one level up, and rename to `NewControllers`... which makes it look nice on call `staticpod.NewControllers(...)`